### PR TITLE
[ASENG-728] Enrich input

### PR DESCRIPTION
**Pull request by d0g0x01**
Various tweaks so support KHaaS and automation workflows:
- Add cluster & runID property to all vertices and store objects
- Enable setting file collector parameter via environment variables
- Add run/cluster support to DSL

### CODE REVIEW

File: configs/etc/kubehound-reference.yaml [hunk 1]
```diff
@@ -29,11 +29,17 @@ collector:
...
   #   # Directory holding the K8s json data files
   #   directory: /path/to/directory
+  #
+  #   # Target cluster name
+  #   cluster: <cluster name>
 
 #
 # General storage configuration
 #
 storage:
```
Comment (edznux-dd) [inline configs/etc/kubehound-reference.yaml:39]: Don't you want to also update the other config files as well? Just to simplify end user life :)

**Review by edznux-dd — COMMENTED** [review]
Do you have the "why we need the run ID" documented somewhere publicly ?
I believe something in a .md here would be useful.

Cluster seems self explanatory but the runs ID isn't (especially what are the implication for the query side: Do we need to provide the ID in all queries now? If we don't are we going to get duplicate entries? Something else?)

**Comment by d0g0x01** [comment]
> Do you have the "why we need the run ID" documented somewhere publicly ? I believe something in a .md here would be useful.
> 
> Cluster seems self explanatory but the runs ID isn't (especially what are the implication for the query side: Do we need to provide the ID in all queries now? If we don't are we going to get duplicate entries? Something else?)

Its for the internal KHaaS use case (sharing/automatic deletion/dashboard/etc), external users can completely ignore so don't think we need to document anything around it

File: configs/etc/kubehound-reference.yaml (hunk 1 above)
Comment (d0g0x01) [reply configs/etc/kubehound-reference.yaml:39]: the other configs don't use the file collector so this field is not required

File: Makefile [hunk 2]
```diff
@@ -127,11 +127,11 @@ test: ## Run the full suite of unit tests
 
 .PHONY: system-test
 system-test: | backend-reset ## Run the system tests
-	cd test/system && export KUBECONFIG=$(ROOT_DIR)/test/setup/${KIND_KUBECONFIG} && go test -v -timeout "60s" -count=1 ./...
+	cd test/system && export KUBECONFIG=$(ROOT_DIR)/test/setup/${KIND_KUBECONFIG} && go test -v -timeout "60s" -count=1 -race ./...
```
[the remark this answers is not in the document]
Comment (d0g0x01) [reply Makefile:130]: this is for system tests only so no mocks, but I did fix the outstanding race condition which had made us disable it so should be reliable now

**Review by edznux-dd — APPROVED** [review]

**Pull request merged**